src/Collection/CollectionTrait.php

```diff
@@ -531,7 +545,7 @@ public function appendItem($item, $key = null)
      */
     public function prepend($items)
     {
-        return (new Collection($items))->append($this);
+        return ($this->newCollection($items))->append($this);
```

Comment:
Extra braces here too which PHP 5.6 does not like.

Reply:
Found one more, both removed